### ISSUE TITLE
Every 10 seconds >> allure_1 | 127.0.0.1 - - [09/Jun/2020 21:39:47] "GET / HTTP/1.1" 200 -

### ISSUE THREAD

**Issue by dE1l** [fescobar/allure-docker-service#82]
Every 10 sends i see in docker logs this massage:
```
allure_1                    | 127.0.0.1 - - [09/Jun/2020 21:39:27] "GET / HTTP/1.1" 200 -
allure_1                    | 127.0.0.1 - - [09/Jun/2020 21:39:37] "GET / HTTP/1.1" 200 -
allure_1                    | 127.0.0.1 - - [09/Jun/2020 21:39:47] "GET / HTTP/1.1" 200 -
allure_1                    | 127.0.0.1 - - [09/Jun/2020 21:39:57] "GET / HTTP/1.1" 200 -
allure_1                    | 127.0.0.1 - - [09/Jun/2020 21:40:07] "GET / HTTP/1.1" 200 -
```
Му docker-compose config:
```
  allure:
    image: "frankescobar/allure-docker-service"
    environment:
      CHECK_RESULTS_EVERY_SECONDS: NONE
      KEEP_HISTORY: "TRUE"
      KEEP_HISTORY_LATEST: 20
      SERVER_URL: "http://localhost:5050/latest-report"
    ports:
      - "5050:5050"
    volumes:
      - ./allure-results:/app/allure-results
      - ./allure-reports:/app/default-reports
```

That is this? How i can switch off this message?

**Comment by fescobar** [comment]
Hi @dE1l 
Those logs are from the API, if you remove that you won't see any logs from the container.
I don't know what is the reason to remove that, because it's from the container.
If you don't want to see any logs at all just run it in the background with `-d` option
```
docker-compose up -d allure
```

**Comment by fescobar** [comment]
Hi @dE1l I've recently re-released the version `2.13.3` (remove previous 2.13.3 in your machine). 
On that version, I included some improvements like hiding log requests.
Thanks
